DhsTranslatorTest/src/test/java/edu/gemini/dhs/translator/test/DhsTranslatorTest.java

```diff
@@ -246,6 +256,43 @@ public void testSetKeywordWrongType() throws ClientProtocolException,
         assertEquals("Operation did not fail.", "error", status.asText());
     }
 
+    @Test
+    public void testConcurrentCalls() throws InterruptedException,
+            ExecutionException {
+        final ConnectionRequester connectionRequester = new ConnectionRequester();
+
+        Callable<String> task = new Callable<String>() {
```

Comment:
Are you used the returned `String` on the checks? Perhaps it should be `Callable<Void>`